src/test/scala/nl/knaw/dans/easy/transform/XmlDdmToCarareSpec.scala

```diff
@@ -120,8 +122,8 @@ class XmlDdmToCarareSpec extends TestSupportFixture with BeforeAndAfterEach {
     (digitalResource \ "appellation" \ "id").text  shouldBe "object.xml"
     (digitalResource \ "description").text  shouldBe "Technical description of the object in xml format"
     (digitalResource \ "format").text  shouldBe "text/xml"
-    (digitalResource \ "link").text  shouldBe "https://download/location/data/object%2Exml"
-    (digitalResource \ "object").text  shouldBe "https://download/location/data/object%2Exml"
+    (digitalResource \ "link").text  shouldBe "https://download/location/12345678-1234-1234-1235-234567890123/data/object%2Exml"
```

Comment:
And in many other locations:
```suggestion
    (digitalResource \ "link").text  shouldBe s"https://download/location/$bagId/data/object%2Exml"
```